[Editor] Improve Cameracontrol in Editor

SVNMLR opened the pull request:
# PR Details

<!--- Provide a general summary of your changes in the Title above -->

- New: Add possibility to change Camera movement speed while pressing RMB (Scroll Mousewheel Up/Down)
- Change: Add more gradations in subdivisions of different movement speeds (not linear)
- Change: Improve UI Layout for Cameraspeed Settings in Sceneview and show the current used value
![New_Cameraspeed_UI-Layout](https://github.com/stride3d/stride/assets/44621949/7c30a428-3950-4887-8a5a-0b2dad25c13f)


- Change: Improve panning speed calculation base on current movement speed
- Change: Enable panning while "Alt"-Key is beeing pressed




<!--- Describe your changes in detail -->

Needs a mention in the Documentation for the new Navigation Controls?!

-> While pressing RMB it is possible to change the movement speed using the Mousewheel.

## Related Issue


## Motivation and Context

At the moment it is not possible to change the movement speed of the Scenecamera while navigation around.

To not have to open the camera menu everytime and dragging the slider to the required movement speed, it is now possible to change it "on the fly" by using the RMB and Scrollwheel while moving around.

It can be annoying beeing blocked on panning by holding "Alt"-Key pressed. 
For example if just pressed "F"-Key to focus on an object and than not beeing able to pan quickly afterwards, until "Alt"-Key was released.

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

manio143 commented on sources/editor/Stride.Assets.Presentation/AssetEditors/EntityHierarchyEditor/Game/EditorGameEntityCameraService.cs:
```diff
@@ -248,7 +261,7 @@ protected void UpdateCameraBase(ref float yaw, ref float pitch, ref Vector3 posi
             {
                 float panningSpeed = asOrthographic ? Component.OrthographicSize : revolutionRadius;
-                panningSpeed *= MouseMoveSpeedFactor * baseSpeed;
+                panningSpeed *= (MouseMoveSpeedFactor * baseSpeed) / 30f;
```
Any reason for this division here? Shouldn't baseSpeed be modified instead?

SVNMLR replied:
The intention here was to reduce the calculated panning speed (depending on the current movement speed) at a constant fraction to have a better feeling. This value was evaluated during testing.

So panning speed should be higher if the movement speed is higher or lower if the movement speed gets lower, too. But not "equal", to not be too fast. Its adapting some fraction of the current choosen movement speed.

manio143 replied:
Ok, since this is called every frame, can you make the division become multiplication instead and extract the constant 1/30 like
```csharp
const float panningSpeedModifier = 1/30f;
```

SVNMLR replied:
I have implemented your mention. Its now a fixed value, which I have predefinde, too.

Additionally I have extended the increments a little further and improved the Camera speed settings in the UI.
It now shows the current speedvalue and the Slider is a little wider, to have better control over the extended increments.

Here you can see the changes to the UI :)
![Camera_Speedsettings](https://github.com/stride3d/stride/assets/44621949/84a46769-47bc-42a9-a3c9-077fbad4ddb7)

I'm done now with all I wanted to do. Hope it is okay now and can be merged :)

manio143 reviewed (approved):
Looks good. Last thing - can you put a screenshot of the updated UI in the PR description?

SVNMLR commented:
> Looks good. Last thing - can you put a screenshot of the updated UI in the PR description?

I have now updated the description and added a screenshot showing the difference :)
